test/setup.sh

```diff
@@ -17,6 +17,7 @@ go get \
   github.com/mattn/goveralls \
   github.com/modocache/gover \
   github.com/tools/godep \
+  github.com/weppos/publicsuffix-go \
```

Comment:
You don't need this part, since the code is vendored. This file only fetches testing tools, not code that is part of the built boulder.